lib/grpc_rest.rb

```diff
@@ -1,6 +1,7 @@
 require 'google/protobuf/well_known_types'
 require 'grpc'
 require 'grpc/core/status_codes'
+require 'interceptors/base_interceptor'
```

Comment:
You can remove this line - you can add to the readme that if you want to make an interceptor you have to require this before you do it.